libvast/vast/fbs/coder.fbs

```diff
@@ -0,0 +1,40 @@
+include "bitmap.fbs";
+
+namespace vast.fbs.coder.detail;
+
+table Base {
+  values: [ulong] (required);
+}
+
+namespace vast.fbs.coder;
+
+table SingletonCoder {
+  bitmap: Bitmap (required);
+}
+
+table VectorCoder {
+  size: ulong;
```

Comment:
Is this really needed? I was wondering whether the size of the list captures this value.

Reply:
It's needed, the values can differ.